card/lib/card/query.rb

```diff
@@ -1,11 +1,27 @@
 # -*- encoding : utf-8 -*-
 
 class Card
+  # Card::Query is for finding implicit lists (or counts of lists) of cards.
+  #
+  # Search and Set cards use Card::Query to query the database, and it's also
+  # frequently used directly in code.
+  #
+  # Query "statements" (objects, really) are made in WQL (Wagn Query
+  # Language). Because WQL is used by Wagneers, the primary language
+  # documentation is on wagn.org. (http://wagn.org/WQL_Syntax). Note that the
+  # examples there are in JSON, like Search card content, but statements in
+  # Card::Query are in ruby form.
+  #
+  # In Wagn's current form, Card::Query generates and executes SQL statements.
+  # However, the SQL generation is largely (not yet fully) separated from the
+  # WQL statement interpretation.
+
```

Comment:
a problem I have with following the code: I don't know what exactly we want to build. An example how the query object representation of an WQL statement looks like could help